graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -397,6 +446,25 @@ def get_selectivity_of_filters_at_vertex(schema_info, filter_infos, parameters,
                     kind=FRACTIONAL_SELECTIVITY, value=fraction_of_domain_queried)
                 selectivity_at_field = _combine_filter_selectivities(
                     [selectivity_at_field, selectivity])
+            elif is_int_field:
+                # int fields are not uniformly distributed but we have quantile information.
+                quantiles = schema_info.statistics.get_field_quantiles(location_name, field_name)
+                if quantiles is not None:
+                    selectivity = Selectivity(
+                        kind=FRACTIONAL_SELECTIVITY,
+                        value=_get_selectivity_fraction_of_interval(
+                            interval,
+                            # Since we can't be sure the minimum observed value is the
+                            # actual minimum value, we treat values less than it as part
+                            # of the first quantile. That's why we drop the minimum and
+                            # maximum observed values from the quantile list.
+                            quantiles[1:-1],
```

Comment:
Why not move this inside `_get_selectivity_fraction_of_interval`?  The function does not what it promises right now.

Reply:
Also, can you add a test when the number of quantiles is less than 2?